external/runtime/Configurations.props

```diff
@@ -4,8 +4,6 @@
     <BuildConfigurations>
       netcoreapp-Windows_NT;
       netcoreapp-Unix;
-      netcoreapp2.0-Windows_NT;
```

Comment:
I don't fully understand why are you getting rid of this. Is there no project that builds for netcoreapp2.0 that has a ReferenceFromRuntime? I find that hard to believe.